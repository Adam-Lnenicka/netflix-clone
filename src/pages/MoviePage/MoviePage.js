import React from "react";
import Navigation from "../../components/FilterNavigation/Navigation";
import Footer from "../../components/Footer/Footer";
import MovieBanner from "../../components/InnerBanner/MovieBanner";
import MovieCardsContainer from "../../components/MovieCardsContainer/MovieCardsContainer";

const MoviePage = () => {
  return (
    <>
      <MovieBanner />
      <Navigation />
      <MovieCardsContainer />
      <Footer />
    </>
  );
};

export default MoviePage;
