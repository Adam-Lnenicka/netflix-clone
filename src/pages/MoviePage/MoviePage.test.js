import React from "react";
import toJSON from "enzyme-to-json";
import { shallow } from "enzyme";
import MoviePage from "./MoviePage";

describe("MoviePage", () => {
  it("Create snapshot of Movie Page", () => {
    const wrapper = shallow(<MoviePage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
