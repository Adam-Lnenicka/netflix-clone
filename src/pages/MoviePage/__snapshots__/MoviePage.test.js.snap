// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePage Create snapshot of Movie Page 1`] = `
<Fragment>
  <MovieBanner />
  <Navigation />
  <ErrorBoundary>
     
    <MovieCardsContainer />
  </ErrorBoundary>
  <Footer />
</Fragment>
`;

exports[`MoviePage create snapshot 1`] = `
<Fragment>
  <MovieBanner />
  <Navigation />
  <ErrorBoundary>
     
    <MovieCardsContainer />
  </ErrorBoundary>
  <Footer />
</Fragment>
`;
